test/configCases/css/pure-css-auto/style.css

```diff
@@ -0,0 +1,39 @@
+@import url("../css-modules/style.module.css");
+
+.class {
+	color: red;
+	background: var(--color);
+}
+
+@keyframes test {
+	0% {
+		color: red;
+	}
+	100% {
+		color: blue;
+	}
+}
+
+:local(.class) {
+	color: red;
+}
+
+:local .class {
+	color: green;
+}
+
+:global(.class) {
+	color: blue;
+}
+
+:global .class {
+	color: white;
+}
+
+:export {
+	foo: bar;
+}
+
+.class {
+	animation: test 1s, test;
```

Comment:
Why you don't use `:local()`?

Reply:
Okay I will try that out with our browsers whether that works, because I only know about local in font-face and not keyframes. Then, I'll update the test case. I'll also test it in production and if that works, I will close this issue. Thank you.